native_client/deepspeech.cc

```diff
@@ -105,13 +112,24 @@ Model::infer(float* aMfcc, int aNFrames, int aFrameLen)
   // Output is an array of shape (1, n_results, result_length).
   // In this case, n_results is also equal to 1.
   auto output_mapped = outputs[0].tensor<int64, 3>();
-  int length = output_mapped.dimension(2) + 1;
-  char* output = (char*)malloc(sizeof(char) * length);
-  for (int i = 0; i < length - 1; i++) {
+  int output_length = output_mapped.dimension(2) + 1;
```

Comment:
nit: I think we should try and use `size_t` for anything related to a dimension.

Reply:
OK.